var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -73,7 +72,7 @@ class Llvm(Package):
             "<current arch>,NVPTX,AMDGPU,CppBackend")
 
     # Build dependency
-    depends_on('cmake@2.8.12.2:', type='build')
+    depends_on('cmake@3.4.3:', type='build')
```

Comment:
Yep, from the `CMakeLists.txt`:
```
cmake_minimum_required(VERSION 3.4.3)
```
That's actually the only reason I touched this package. But I figured I should update it to `CMakePackage` while I was at it.